src/screens/People/People.js

```diff
@@ -151,6 +195,10 @@ type Props = {
   theme: Theme,
   referralsFeatureEnabled: boolean,
   goToInvitationFlow: () => void,
+  sendInvitation: Function,
+  acceptInvitation: Function,
+  cancelInvitation: Function,
+  rejectInvitation: Function,
```

Comment:
Can we have something more descriptive than `Function` in these? `(user: ApiUser) => void` / `(invitation: Object) => void`.